### PR TITLE
Calculate accuracy of runtime

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

feature

## Description

- add script of calculating diff
- add script of calculating accuracy

usage
```
python scripts/cal_acc.py --model_name ~/models/llama/Llama-2-7b-chat-hf/ --model_format runtime --tasks piqa
```

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: neural_speed/application/main_pybind.cpp [hunk 1]
```diff
@@ -73,7 +73,8 @@ class Model {
   // deprecated API
   std::vector<std::vector<model_token>> generate_tokens(const std::vector<std::vector<model_token>>& input_ids);
   const std::vector<float>& evaluate_(const std::vector<std::vector<model_token>>& input_ids);
-  py::array_t<float> evaluate(const std::vector<std::vector<model_token>>& input_ids) {
+  py::array_t<float> evaluate(const std::vector<std::vector<model_token>>& input_ids, bool logits_all = false) {
+    if (logits_all) ctx->logits_all = true;
```
Comment (DDEle) [inline neural_speed/application/main_pybind.cpp:77]: Otherwise, it seems to be an unexpected behavior if called `evaluate` with `logits_all = true` and then `logits_all = false`
```suggestion
    ctx->logits_all = logits_all;
```

Comment (zhenwei-intel) [reply neural_speed/application/main_pybind.cpp]: thx

**Comment by a32543254** [comment]
add reuqirment.txt

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by a32543254 — COMMENTED** [review]
Could we add extension test for acc test

**Comment by VincyZhang** [comment]
> Could we add extension test for acc test
We will later.
Please provide your local result first @zhenwei-intel

**Review by VincyZhang — APPROVED** [review]

**Comment by zhenwei-intel** [comment]
> > Could we add extension test for acc test
> > We will later.
> > Please provide your local result first @zhenwei-intel

updated
